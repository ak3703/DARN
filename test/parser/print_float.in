int main() {
	float x;
	x = 4.0;
	printf(x);

}