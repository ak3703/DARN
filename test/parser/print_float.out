
int main()
{
float x;
x =  4.;
printf(x);
} 
